tools/bin/check-twig-templates-syntax

```diff
@@ -36,7 +36,11 @@
 
 define('GLPI_ROOT', dirname(__DIR__, 2));
 
-include_once(GLPI_ROOT . '/inc/based_config.php');
+require_once GLPI_ROOT . '/vendor/autoload.php';
+
+$env = $_ENV['GLPI_ENVIRONMENT_TYPE'] ?? 'development';
```

Comment:
No need to make it configurable here. The `tools` directory is only used by developpers/CI.
```suggestion
$env = 'development';
```